services/gateway/apis/http-version2/methods/transactionsStatisticsDay.js

```diff
@@ -25,6 +26,24 @@ module.exports = {
 		offset: { optional: true, type: 'number', default: 0, min: 0 },
 		limit: { optional: true, type: 'number', default: 10, min: 1, max: 100 },
 	},
+	get schema() {
+		const transactionSchema = {};
+		transactionSchema[this.swaggerApiPath] = { get: {} };
+		transactionSchema[this.swaggerApiPath].get.tags = this.tags;
+		transactionSchema[this.swaggerApiPath].get.summary = 'Requests transaction statistics day';
+		transactionSchema[this.swaggerApiPath].get.description = `Returns transaction statistics day\n RPC=> ${this.rpcMethod}`;
+		transactionSchema[this.swaggerApiPath].get.parameters = transformParams('transactions', this.params);
```

Comment:
Could you please confirm if the input param to `transformParams` should be `transactions` or different?

Reply:
Name of Input param doesn't really matter in this case, we can use nay name. I only used `transactions` as it belong to transactions tag.